### ISSUE TITLE
Edge-Scrolling activation gets lost (T8418)

### ISSUE THREAD

**Issue by celticmagic** [getsolus/packages#249]
<div dir="rtl"><b>Frank&nbsp;Röber</b>&nbsp;(<em>&#35;roebotron</em>),&nbsp;2019-10-14&nbsp;18:38:24&nbsp;UTC</div><br/>

Every time I open Settings/Devices/Mouse & Touchpad, the setting "Edge Scrolling" is OFF.
I can activate it and then it works fine until re-opening the settings dialog. Then it falls back to OFF.

Laptop FJSC Esprimo Mobile U9210, Solus 4.0/4.2 64bit with Budgie

**Comment by roebotron** [comment]
I just did a check: This issue still persists.

**Comment by EbonJaeger** [comment]
I wonder... If you open `budgie-control-center` in the terminal, go to the panel, and turn the option on, does any error appear in the terminal?

**Comment by roebotron** [comment]
> I wonder... If you open `budgie-control-center` in the terminal, go to the panel, and turn the option on, does any error appear in the terminal?

No, no output at all. It's a pretty simple effect: Turn option on, change to another panel, come back and it's turned off. Like a hard initialization or an error in reading the current (last written) state.

**Comment by roebotron** [comment]
May this information from dmesg help?
psmouse serio4: synaptics: queried max coordinates: x [..5756], y [..5088]
psmouse serio4: synaptics: Touchpad model: 1, fw: 7.0, id: 0x1c0b1, caps: 0xd04791/0xb02000/0x20000/0x0, board id: 0, fw id: 510125
psmouse serio4: synaptics: serio: Synaptics pass-through port at isa0060/serio4/input0
input: SynPS/2 Synaptics TouchPad as /devices/platform/i8042/serio4/input/input13

**Comment by TraceyC77** [comment]
Budgie 10.10 is in testing for Solus, which will be Wayland and use a different toolkit. Please open a new bug report if this issue is still present on that version. Thanks.
